backend/fixtures/postgres/sql/job_offers.sql

```diff
@@ -0,0 +1,10 @@
+insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (7, 'ykose6@indiegogo.com', 'Skajo', 7, false, 'Nam dui. Proin leo odio, porttitor id, consequat in, consequat ut, nulla.', 'Integer ac neque.', 'https://nationalgeographic.com/potenti/cras/in/purus/eu.xml', 'msplevings6@prweb.com', '8/10/2022', '10/16/2022');
+insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (8, 'bnorker7@mit.edu', 'Camido', 8, true, 'Aenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh. Quisque id justo sit amet sapien dignissim vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla dapibus dolor vel est. Donec odio justo, sollicitudin ut, suscipit a, feugiat et, eros. Vestibulum ac est lacinia nisi venenatis tristique. Fusce congue, diam id ornare imperdiet, sapien urna pretium nisl, ut volutpat sapien arcu sed augue.', 'Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor.', 'https://cdc.gov/varius/integer/ac/leo/pellentesque/ultrices.html', 'mruppertz7@ed.gov', '8/20/2022', '12/13/2022');
+insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (9, 'kbattill8@chron.com', 'Camimbo', 9, true, 'Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante. Nulla justo. Aliquam quis turpis eget elit sodales scelerisque. Mauris sit amet eros. Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor. Duis mattis egestas metus.', 'Nulla ac enim. In tempor, turpis nec euismod scelerisque, quam turpis adipiscing lorem, vitae mattis nibh ligula nec sem.', 'http://reuters.com/commodo/placerat/praesent/blandit/nam/nulla/integer.png', 'ggarrigan8@nbcnews.com', '8/28/2022', '2/14/2023');
+insert into job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat) values (10, 'cbinfield9@github.com', 'Zoombox', 10, false, 'In hac habitasse platea dictumst. Maecenas ut massa quis augue luctus tincidunt. Nulla mollis molestie lorem. Quisque ut erat. Curabitur gravida nisi at nibh. In hac habitasse platea dictumst. Aliquam augue quam, sollicitudin vitae, consectetuer eget, rutrum at, lorem. Integer tincidunt ante vel ipsum. Praesent blandit lacinia erat.', 'Nulla tellus. In sagittis dui vel nisl. Duis ac nibh.', 'http://salon.com/quam/nec.html', 'sbenoit9@example.com', '8/20/2022', '5/13/2022');
```

Comment:
```suggestion
INSERT INTO job_offers (id, email, company_name, title_id, is_remote, description, how_to_apply, apply_url, apply_email_address, createdat, updatedat)
VALUES (1, 'jgittoes0@g.co', 'Yombu', 1, FALSE, 'Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Etiam vel augue. Vestibulum rutrum rutrum neque.', 'Morbi quis tortor id nulla ultrices aliquet. Maecenas leo odio, condimentum id, luctus nec, molestie sed, justo.', 'https://about.me/nulla/ultrices/aliquet/maecenas/leo.js', 'owhitton0@seesaa.net', '5/10/2022', '4/22/2022'),
       (2, 'mcuskery1@seattletimes.com', 'Youopia', 2, FALSE, 'Cras pellentesque volutpat dui. Maecenas tristique, est et tempus semper, est quam pharetra magna, ac consequat metus sapien ut nunc. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Mauris viverra diam vitae quam. Suspendisse potenti. Nullam porttitor lacus at turpis. Donec posuere metus vitae ipsum. Aliquam non mauris. Morbi non lectus. Aliquam sit amet diam in magna bibendum imperdiet.', 'Curabitur gravida nisi at nibh. In hac habitasse platea dictumst.', 'https://abc.net.au/adipiscing/molestie/hendrerit/at.aspx', 'ravramchik1@china.com.cn', '5/16/2022', '10/2/2022'),
       (3, 'djerok2@fema.gov', 'DabZ', 3, FALSE, 'Nunc nisl. Duis bibendum, felis sed interdum venenatis, turpis enim blandit mi, in porttitor pede justo eu massa. Donec dapibus.', 'In eleifend quam a odio.', 'https://wordpress.com/lectus/pellentesque/eget/nunc/donec/quis.json', 'simm2@bbb.org', '7/15/2022', '12/16/2022'),
       (4, 'gpelchat3@dropbox.com', 'Oodoo', 4, FALSE, 'Suspendisse potenti. Nullam porttitor lacus at turpis. Donec posuere metus vitae ipsum. Aliquam non mauris.', 'Duis aliquam convallis nunc. Proin at turpis a pede posuere nonummy. Integer non velit.', 'https://zimbio.com/ultrices.aspx', 'adouce3@oracle.com', '5/31/2022', '4/23/2022'),
       (5, 'anelthorpe4@yandex.ru', 'Rhyzio', 5, FALSE, 'Duis bibendum. Morbi non quam nec dui luctus rutrum. Nulla tellus. In sagittis dui vel nisl. Duis ac nibh. Fusce lacus purus, aliquet at, feugiat non, pretium quis, lectus. Suspendisse potenti. In eleifend quam a odio. In hac habitasse platea dictumst. Maecenas ut massa quis augue luctus tincidunt.', 'Integer a nibh. In quis justo.', 'https://loc.gov/vulputate/justo/in/blandit.js', 'sginni4@toplist.cz', '1/8/2023', '12/3/2022'),
       (6, 'kcleaver5@yolasite.com', 'Abata', 6, FALSE, 'Praesent blandit. Nam nulla. Integer pede justo, lacinia eget, tincidunt eget, tempus vel, pede. Morbi porttitor lorem id ligula. Suspendisse ornare consequat lectus. In est risus, auctor sed, tristique in, tempus sit amet, sem. Fusce consequat. Nulla nisl. Nunc nisl. Duis bibendum, felis sed interdum venenatis, turpis enim blandit mi, in porttitor pede justo eu massa.', 'Integer non velit. Donec diam neque, vestibulum eget, vulputate ut, ultrices vel, augue.', 'https://pen.io/non/mattis/pulvinar/nulla/pede/ullamcorper.json', 'pashbee5@reference.com', '11/9/2022', '1/25/2023'),
       (7, 'ykose6@indiegogo.com', 'Skajo', 7, FALSE, 'Nam dui. Proin leo odio, porttitor id, consequat in, consequat ut, nulla.', 'Integer ac neque.', 'https://nationalgeographic.com/potenti/cras/in/purus/eu.xml', 'msplevings6@prweb.com', '8/10/2022', '10/16/2022'),
       (8, 'bnorker7@mit.edu', 'Camido', 8, TRUE, 'Aenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh. Quisque id justo sit amet sapien dignissim vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla dapibus dolor vel est. Donec odio justo, sollicitudin ut, suscipit a, feugiat et, eros. Vestibulum ac est lacinia nisi venenatis tristique. Fusce congue, diam id ornare imperdiet, sapien urna pretium nisl, ut volutpat sapien arcu sed augue.', 'Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor.', 'https://cdc.gov/varius/integer/ac/leo/pellentesque/ultrices.html', 'mruppertz7@ed.gov', '8/20/2022', '12/13/2022'),
       (9, 'kbattill8@chron.com', 'Camimbo', 9, TRUE, 'Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante. Nulla justo. Aliquam quis turpis eget elit sodales scelerisque. Mauris sit amet eros. Suspendisse accumsan tortor quis turpis. Sed ante. Vivamus tortor. Duis mattis egestas metus.', 'Nulla ac enim. In tempor, turpis nec euismod scelerisque, quam turpis adipiscing lorem, vitae mattis nibh ligula nec sem.', 'http://reuters.com/commodo/placerat/praesent/blandit/nam/nulla/integer.png', 'ggarrigan8@nbcnews.com', '8/28/2022', '2/14/2023'),
       (10, 'cbinfield9@github.com', 'Zoombox', 10, FALSE, 'In hac habitasse platea dictumst. Maecenas ut massa quis augue luctus tincidunt. Nulla mollis molestie lorem. Quisque ut erat. Curabitur gravida nisi at nibh. In hac habitasse platea dictumst. Aliquam augue quam, sollicitudin vitae, consectetuer eget, rutrum at, lorem. Integer tincidunt ante vel ipsum. Praesent blandit lacinia erat.', 'Nulla tellus. In sagittis dui vel nisl. Duis ac nibh.', 'http://salon.com/quam/nec.html', 'sbenoit9@example.com', '8/20/2022', '5/13/2022');
```

- more readable IMO
- one query for bulk insert instead of 10 queries

What do you think ?